tests/phpunit/Metrics/TargetDetectionStatusesProviderTest.php

```diff
@@ -101,9 +101,9 @@ public function test_it_ignores_some_statuses_when_debugging_is_not_enabled_for_
         $provider = new TargetDetectionStatusesProvider($logs, LogVerbosity::NORMAL, false, false);
 
         $this->assertProvidesExcluding([
-                DetectionStatus::KILLED,
-                DetectionStatus::ERROR,
-            ],
+            DetectionStatus::KILLED,
+            DetectionStatus::ERROR,
+        ],
```

Comment:
looks worse, but as I understand we can't do anything?

Reply:
Yeah that's the style I pretty much go to by default, I find it preferable over semi on the same line and semi on a new line

Reply:
Will address them separately.